src/types.ts

```diff
@@ -72,6 +72,12 @@ export type WaldoCapabilityOptions = WaldoSharedOptions & {
     /** Name of the session that will appear in Waldo session history */
     displayName?: string;
 
+    /** Language of the session */
+    language?: string;
```

Comment:
We can add it, but we currently didn't document all capabilities where we just duplicate appium ones (appActivity, appPackage, appWaitActivity, bundleId, app) except for deviceName and automationName.

So maybe we should add theses too.

Reply:
I removed it 👍